### DISCUSSION TITLE
Why are holidays that are not adopted in whole of UK not in their subdivision only?

### DISCUSSION THREAD

**Discussion by ShutterX** [vacanza/holidays#1307] [Q&A]
These holidays are added through _add_subdiv_uk_holidays and have the country name in [ ] as a suffix of the holiday name.
Example: St. Patricks day is under UK as: "St. Patrick's Day [Northern Ireland]" & under Northern Ireland subdiv as: "St. Patrick's Day"

**Comment by arkid15r** [comment]
Hi @ShutterX,
thank you for starting this discussion.

I guess the best answer to your question would be "the original author implemented it that way". The `united_kingdom.py` has recently been refactored by @KJhellico as a part of our standardization process. The user facing part was kept unchanged intentionally. I believe this behavior will be revised based on your question (no ETA though).

Let me know in case you have any questions or suggestions.

Thank you!

**Reply by PPsyrius** [reply]
AFAIK this is now fixed as of 0.27 release

**Comment by arkid15r** [comment]
Right, this was addressed in https://github.com/dr-prodigy/python-holidays/pull/1316
